olm_deploy/operatorregistry/registry-deployment.yaml

```diff
@@ -18,7 +18,7 @@ spec:
       initContainers:
       - name: mutate-csv-and-generate-sqlite-db
         image: ${IMAGE_CLUSTER_LOGGING_OPERATOR_REGISTRY}
-        imagePullPolicy: IfNotPresent
+        imagePullPolicy: Always
```

Comment:
I don't think we want Always here unconditionally - I can see why it's helpful in development but it's expensive/fragile in a production system. I suggest using the short git commit hash as part of the image name - that way the image name will always be different if it was built from different code and there's no need for Always.

Reply:
/hold

Reply:
using git hash in development may not that useful, a readable branch name is more effective.
moreover this imagePullPolicy was already set to `Always` , i had changed it to `IfNotPresent` in earlier PR

Reply:
in development we use env variables to set up the images to be used in logging deployment `IMAGE_CLUSTER_LOGGING_OPERATOR` , setting an env variable with git hash in the image name will not be developer friendly

Reply:
@alanconway does this address your concerns

Reply:
@vimalk78 Yep, it makes sense for development. 
Not sure why we have Always but it sounds like that is a separate question from this PR.